civihr_default_theme/scss/components/_header.scss

```diff
@@ -0,0 +1,299 @@
+      outline: none;
+    }
+  }
+
+  > li {
+    border-bottom: 1px solid $gray-light;
+    padding-left: 20px;
+
+    &:last-child {
+      border: none;
+    }
+  }
+
+  @media(min-width: $screen-sm-min) {
+    a {
+      color: inherit;
+      padding: 0 15px;
+
+      &.active,
+      &:hover,
+      &:focus {
+        background: $gray;
+        color: $light;
+      }
+    }
+
+    > li {
+      border: none;
+      padding: 0;
+      float: left;
+    }
+  }
+}
+
+.chr_header__nav__toggle {
+  cursor: pointer;
+  display: table-cell;
+  height: 100%;
+  line-height: normal;
+  vertical-align: middle;
+}
+
+
+/**
+ * User menu in main header <nav>
+ */
+.chr_header__user-menu {
+  color: $light-pale;
+  cursor: default;
+  padding: 0 10px;
+  position: relative;
+
+  &:hover {
+    background: $crm-gray-darker-alt;
+    color: $light;
+
+    > .chr_header__sub-menu {
+      display: block;
+    }
+  }
+
+  .chr_header__sub-menu {
+    right: 0;
+  }
+}
+
+.chr_header__user-menu__data {
+
+  > .chr_header__user-menu__name {
+    float: left;
+  }
+
+  > .chr_profile-card {
+    margin: -2px 8px 0 8px; // To improve, the vertical margin is just to vertically align the card
+    vertical-align: top;
+  }
+
+  @media(max-width: $screen-md-min) {
+    > .chr_header__user-menu__arrow,
+    > .chr_header__user-menu__name {
+      display: none;
+    }
+  }
+}
+
+.chr_header__settings-menu {
+  position: relative;
+  display: inline-block;
+  width: 50px;
+  height: 50px;
+  text-align: center;
+  float: right;
+
+  &:hover {
+    background-color: $crm-gray-darker-alt;
+
+    > .chr_header__sub-menu {
+      display: block;
+    }
+  }
+
+  > .fa-cog {
+    font-size: 18px;
+  }
+
+  .chr_header__sub-menu {
+    right: 0;
+  }
+}
+
+.chr_header__home-menu {
+
+  &:hover {
+    > .chr_header__sub-menu {
+      display: block;
+    }
+  }
+
+  .chr_header__sub-menu {
+    left: 0;
+  }
+}
+
+.chr_header__sub-menu {
+  background: $crm-gray-darker-alt;
+  display: none;
+  position: absolute;
+  z-index: 99;
+
+  li {
+    text-align: left;
+  }
+
+  a {
+    color: $light-pale;
+    font-size: 1.3rem;
+    display: block;
+    padding: 0 30px 0 20px;
+    white-space: nowrap;
+
+    &:hover {
+      background: $crm-brand-secondary;
+      color: $light;
+    }
+
+    > i {
+      margin-right: 10px;
+    }
+  }
+}
+
+//CiviHR menu in small devices
+.chr_header__brand .chr_header__home-sub-menu {
+  right: 0;
+  width: 120px;
+  margin: auto;
+}
+
+
+@media (max-width: 767px) {
+  .chr_header__nav__menu > li {
+    border-bottom: none;
+    padding: 0;
+  }
+
+  .chr_header__nav__menu > li > a {
+    border-bottom: 1px solid $crm-grizzly;
+    margin: 0 25px;
+    padding: 0 20px;
+  }
+
+  .chr_header__nav__menu > li:last-child > a {
+    border-bottom: none;
+  }
+
+  .chr_header__nav.is-open {
+    padding: 0px
+  }
+}
+
+
+//Hide Menus when not logged in
+.not-logged-in {
+  .chr_header__home-menu .chr_header__sub-menu {
```

Comment:
We are already hiding some menu's in menu.scss. This should be clubbed with that code.